code/controllers/subsystems/migration.dm

```diff
@@ -18,7 +18,7 @@ SUBSYSTEM_DEF(migration)
 	wait = 300 //Ticks once per 30 seconds
 
 	var/burrow_scan_interval = 5 MINUTES //Every 5 minutes, they'll scan and catalogue the lifeforms around them
-	var/burrow_plantspread_interval = 1 MINUTES //Every 10 minutes, plants near burrows will spread through them
```

Comment:
i changed this to 1 minute for debugging once, and forgot to change it back. explains why theres been so much spam in the logs